src/react-components/src/components/02-molecules/menus/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -22,16 +26,28 @@ export default class Breadcrumb extends React.Component<BreadcrumbProps, {}> {
 
     let bcrumbItems = [];
 
-    breadcrumbs.forEach((item: breadcrumb, index: number) => {
+    breadcrumbs.forEach((item: breadcrumb|JSX.Element, index: number) => {
       let last = index === breadcrumbs.length - 1;
       if (last) {
         // TODO: SFR-716 Incorporate Icon
       }
-      let linkContent: string | JSX.Element = item.url ? <a href={item.url} className={bem("link", [], breadcrumbs__base_class)}>{item.text}</a> : item.text;
-
+      let linkContent: string | JSX.Element;
+      if (isTextBreadcrumb(item)) {
+        linkContent = item.url ? <a href={item.url} className={bem("link", [], breadcrumbs__base_class)}>{item.text}</a> : item.text;
+      } else {
+        let props = {
+          className: bem("link", [], breadcrumbs__base_class),
+        };
+        let blah = React.createElement(
```

Comment:
This is incredibly generalized, I think after our `Link` work I will make it more specific.  
Also, I should rename this var

Reply:
`let passedComponent`?